tests/functional/spec/components/radio/radio_visible_answers.spec.js

```diff
@@ -0,0 +1,28 @@
+const RadioVisibleTruePage = require('../../../generated_pages/radio_detail_answer_visible/radio-visible-true.page.js');
+const RadioVisibleFalsePage = require('../../../generated_pages/radio_detail_answer_visible/radio-visible-false.page.js');
+const RadioVisibleNonePage = require('../../../generated_pages/radio_detail_answer_visible/radio-visible-none.page.js');
+
+describe('Given I start a Radio survey with a write-in option', function() {
+  beforeEach(function() {
+    browser.openQuestionnaire('test_radio_detail_answer_visible.json');
+  });
+
+  it('When I view a write-in radio and the visible option is set to true, Then the detail answer label should be displayed', function() {
+    $(RadioVisibleTruePage.coffee()).click();
+    expect($(RadioVisibleTruePage.otherDetail()).isDisplayed()).to.equal(true);
+  });
```

Comment:
The other detail is displayed by default, so shouldn't need a click prior to this check?

Reply:
Is this testing the default behaviour, or behaviour after a click?

If this test is actually testing that the visible write in doesn't collapse after the coffee option is selected, there should probably be an additional test with an appropriate description. The click step then may be removed from the default behaviour test here.

Reply:
Fixed the first test, added new one to cover "visible write in doesn't collapse" behaviour, thanks!